README.md

```diff
@@ -31,8 +26,8 @@ In addition to the previous instructions, you will need to follow these steps to
     Enter your heroku credentials when prompted.
     _A free heroku account can be created at https://heroku.com_
 - Add your public key so that you can deploy code to your heroku apps
-    ``` heroku keys:add ``` 
-    and enter 'y' to add the key. 
+    ``` heroku keys:add ```
+    and enter 'y' to add the key.
```

Comment:
💁 My editor removes an extra white space when I save a file, so this is just white space clean up.